packages/react-components/src/templates/__tests__/UserProfileHeader.test.tsx

```diff
@@ -111,6 +111,29 @@ describe('an edit button', () => {
   });
 });
 
+it('is rendered for avatar', () => {
+  const { getByLabelText } = render(
+    <UserProfileHeader
+      {...boilerplateProps}
+      onImageSelect={(file: File) => {}}
+    />,
+  );
+  expect(getByLabelText(/edit.+avatar/i)).toBeVisible();
+  expect(getByLabelText(/upload.+avatar/i)).not.toHaveAttribute('disabled');
+});
+
+it('is disabled for avatar (REGRESSION)', () => {
+  disable('EDIT_PROFILE_REST');
```

Comment:
We probably want a separate flag for this? `REST` is already active on prod and this shouldn't go live without error handling